docs/source/blocks/blocks-core/teaser.md

```diff
@@ -0,0 +1,152 @@
+---
+myst:
+  html_meta:
+    "description": "Volto Teaser Block description and developer notes"
+    "property=og:description": "Volto Teaser Block description and developer notes"
+    "property=og:title": "Volto Teaser Block"
+    "keywords": "Volto, React, blocks, teaser, Plone"
+---
+
+(teaser-block)=
+
+# Teaser Block
+
+Volto comes with a Teaser block that given a content object as target is able to pull information from it and show it in the block.
```

Comment:
```suggestion
Volto comes with a Teaser block, which when given a content object as a target, is able to pull information from the targeted content object and show it in the block.
```

Reply:
This one was outdated already, modified by other amendment.

Reply:
Nope. I started another review. Please stand by.